zhaquirks/tuya/ts0601_switch.py

```diff
@@ -127,7 +128,6 @@ class TuyaSingleSwitch_GP(TuyaSwitch):
         MODELS_INFO: [
             ("_TZE200_gbagoilo", "TS0601"),  # reported in #1634
             ("_TZE204_6fk3gewc", "TS0601"),
-            ("_TZE204_ptaqh9tk", "TS0601"),  # reported in #2780
```

Comment:
Don't remove the model declaration here.
It can be defined in both classes.

Reply:
Ready